txscript/data/script_tests.json

```diff
@@ -1238,9 +1238,112 @@
 ["2147483648 1", "ROTL 1 EQUAL", "NONE", "ERR_OUT_OF_RANGE", "ROTL must fail with input value >4 bytes"],
 ["1 2147483648", "ROTL TRUE", "NONE", "ERR_OUT_OF_RANGE", "ROTL must fail with rotation count >4 bytes"],
 
-["2 DUP MUL", "4 EQUAL", "NONE", "OK"],
+["NUMERIC opcode tests"],
 
-["Division related test coverage"],
+["1ADD test coverage"],
+["NOP", "1ADD 1", "NONE", "ERR_INVALID_STACK_OPERATION", "1ADD requires one item on the stack and NOP must not be treated as one"],
+["", "1ADD 1", "NONE", "ERR_INVALID_STACK_OPERATION", "1ADD requires one item on the stack, stack has no items"],
+["1 1ADD", "2 EQUAL", "NONE", "OK"],
+["2147483647", "1ADD 2147483648 EQUAL", "NONE", "OK", "We can do math on 4-byte integers, and compare 5-byte ones"],
+["2147483647", "1ADD 1", "NONE", "OK"],
+["-2147483647", "1ADD 1", "NONE", "OK"],
+["2147483648 1ADD", "1", "NONE", "ERR_OUT_OF_RANGE", "1ADD must fail with stack item >4 bytes"],
+["2147483647", "1ADD 1SUB 1", "NONE", "ERR_OUT_OF_RANGE", "We cannot do math on 5-byte integers, even if the result is 4-bytes"],
+["0x7a{6}", "1ADD 1", "NONE", "ERR_INVALID_STACK_OPERATION", "1ADD must fail with a non-numeric stack item"],
```

Comment:
This, and all of the tests like it, aren't actually testing what the comment says.  `0x7a` is not a data push operation, rather it is `OP_ROLL`, so the failure is because there are no inputs to `OP_ROLL`.

Since numerbers are just raw bytes popped from the stack and interpreted as numerics according to the script number rules, it's not actually possible to test non-numeric inputs since they are all effectively numeric.  That edge conditions, such as max 4-bytes, which the above test already handles, are the relevant tests.

Consequently, this test, and all similar, should be removed.